src/applications/vaos/tests/e2e/page-objects/ClinicChoicePageObject.js

```diff
@@ -17,13 +16,15 @@ export class ClinicChoicePageObject extends PageObject {
   selectClinic({ selection, isCovid = false } = {}) {
     if (isCovid) {
       cy.findByText(/Choose a clinic/i, { selector: 'h1' });
+      cy.findByLabelText(selection).as('radio');
+      cy.get('@radio').check();
     } else {
-      cy.findByText(/Choose a VA clinic/i, { selector: 'h1' });
+      cy.get('va-radio')
+        .shadow()
+        .get(`va-radio-option[value=${selection}]`)
```

Comment:
A little odd but it seems that selection by text works inconsistently with VARadioField so I'm opting to select by value instead, which works more consistently.

Reply:
I would look into this a little more since checking by value is an implementation detail. It's recommended to test what is seen on the screen.

Reply:
Hmm okay let me try a few more things.